src/main/java/org/wso2/carbon/hash/provider/pbkdf2/ErrorMessage.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.hash.provider.pbkdf2;
+
+/**
+ * The ENUM includes all the error messages of hashing.
+ */
+public enum ErrorMessage {
+
+    // Client Errors.
+    ERROR_CODE_EMPTY_VALUE("60001", "Empty value", "Value cannot be empty"),
+
+    // Server Errors.
+    ERROR_CODE_NO_SUCH_ALGORITHM("60501", "Pseudo Random Function was not detected",
```

Comment:
This should be not supported PRF right? Shall we modify the description and the message according to that? Also, lets log the PRF as well.